src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
```

Comment:
- [ ] **(OPTIONAL)** You can destructure this `name` props directly in the argument part of the functional component, like `const Button = ({ name }) => {` for example :smile: